src/view/components/AddEntry/index.tsx

```diff
@@ -0,0 +1,195 @@
+import * as React from 'react';
+import { Content, List, ListItem, Text, Grid, Row, Col, H1, Button, Input, Form, Item, Label } from 'native-base';
+import * as moment from 'moment';
+
+import { Farmer } from '../../../store/modules/farmer/types';
+import { MilkEntry } from '../../../store/modules/milk/types';
+
+import { Route } from '../../navigation/navigator';
+import { MapStateToProps, MapDispatchToProps, connect } from 'react-redux';
+import navActions from '../../../store/modules/nav/actions';
+import Composer from '../../hoc/PageComposer/index';
+import { State } from '../../../store/types';
+import milkThunks from '../../../store/modules/milk/thunks';
+
+import Styles from './style';
+
+interface OwnPropsType {
+}
+
+interface DispatchPropsType {
+  navigate(route: Route): void;
+  goBack(): void;
+  createMilkEntry(newEntry: MilkEntry): void;
+}
+
+interface StorePropsType {
+  farmer: Farmer;
+}
+
+/** AddEntry PropsType */
+type PropsType = StorePropsType & DispatchPropsType & OwnPropsType;
+
+interface OwnStateType {
+  amountOfProduct: number;
+  quality: string;
```

Comment:
I wonder if we should make this number since it's a measured value (I don't know the units though)

Reply:
Are we sure it's a number though?